rosdep/python.yaml

```
@@ -2239,6 +2239,8 @@ python-ipdb:
   debian: [python-ipdb]
   gentoo: [dev-python/ipdb]
   ubuntu: [python-ipdb]
+python-is-python3:
+  ubuntu: [python-is-python3]
```

Since this is only available since focal, and we still support older distributions, we would need this to be:

```suggestion
  ubuntu: 
    '*': [python-is-python3]
    bionic: null
    xenial: null
```